daml.yaml

```
@@ -4,7 +4,7 @@
 #
 
 sdk-version: 0.13.29
-scenario: Tests.MortgagePoolAgreementDraftTests:addPoolToUnderlyingAllosUsToAddMultiplePools
+scenario: Main.Main:populate
 name: mortgage-repackaging
 source: daml/LibraryModules.daml
```

Could you make this change?
```suggestion
source: daml
```

For reasoning please, see this thread: https://daholdings.slack.com/archives/CG2UVAT1N/p1570800271009500